share/picongpu/examples/FieldAbsorberTest/cmakeFlags

```diff
@@ -38,58 +38,60 @@ flags[3]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=ArbitraryOrderFDTD< 4 MY_
 # Test that PML compiles in 3D and 2D
 flags[4]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=YeePML< CurrentInterpolation >;-DPARAM_PML_SIZE=8;-DPARAM_PML_KAPPA_MAX=1.0;-DPARAM_PML_ALPHA_MAX=0.2'"
 flags[5]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=YeePML< CurrentInterpolation >;-DPARAM_PML_SIZE=8;-DPARAM_PML_KAPPA_MAX=1.0;-DPARAM_PML_ALPHA_MAX=0.2;-DPARAM_DIMENSION=DIM2'"
-
+# Test that arbitrary-order solver with PML absorbing boundary conditions compiles in 3D and 2D
+flags[6]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=ArbitraryOrderFDTDPML< 4 MY_AOFDTD_COMMA CurrentInterpolation >;-DPARAM_PML_SIZE=8;-DPARAM_PML_KAPPA_MAX=1.0;-DPARAM_PML_ALPHA_MAX=0.2'"
+flags[7]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=ArbitraryOrderFDTDPML< 4 MY_AOFDTD_COMMA CurrentInterpolation >;-DPARAM_PML_SIZE=8;-DPARAM_PML_KAPPA_MAX=1.0;-DPARAM_PML_ALPHA_MAX=0.2;-DPARAM_DIMENSION=DIM2'"
 
```

Comment:
I am not sure if a custom macro `MY_AOFDTD_COMMA` for comma is good. What do you think @psychocoderHPC ?

Reply:
There is also a boost PP comma available.

Reply:
Refactored to use the boost comma.